### PR TITLE
Files Binary ACE

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

**Review by vincenttran-msft — APPROVED** [review]
:shipit:

**Comment by seanmcc-msft** [comment]
TODO I think I forgot to add support for this in Create Directory.

**Review by nickliu-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs [hunk 1]
```diff
@@ -3456,6 +3601,50 @@ private void SetNameFieldsIfNull()
...
         #region CreateDirectory
+        /// <summary>
+        /// The <see cref="CreateDirectory"/> operation creates a new
+        /// directory in this share.
+        ///
+        /// For more information, see
+        /// <see href="https://docs.microsoft.com/rest/api/storageservices/create-directory">
+        /// Create Directory</see>.
+        /// </summary>
+        /// <param name="directoryName">T
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs:3612]: NIT - remove typo of `T at the end of this line

File: sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs [hunk 2]
```diff
@@ -3488,25 +3677,73 @@ private void SetNameFieldsIfNull()
...
 
+        /// <summary>
+        /// The <see cref="CreateDirectoryAsync"/> operation creates a new
+        /// directory in this share.
+        ///
+        /// For more information, see
+        /// <see href="https://docs.microsoft.com/rest/api/storageservices/create-directory">
+        /// Create Directory</see>.
+        /// </summary>
+        /// <param name="directoryName">T
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs:3711]: NIT - same here

**Review by amnguye — APPROVED** [review]
LGTM - just had some nit picky comments

File: sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs]: Fixed.

File: sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs]: Fixed.

**Pull request merged**